magento/adobe-stock-integration
# #576 : Licensed overlay labels styling to match mockups

### Description (*)

This PR will introduce the styling of the licensed overlay label changes

### Fixed Issues (if relevant)

1. https://github.com/magento/adobe-stock-integration/issues/576: Licensed overlay labels styling to match mockups

### Manual testing scenarios (*)
1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Log in to Adobe Stock (previously licensed images should be marked as "Licensed" in the grid)
5. Find images licensed for the account used to login and see "Licensed" labels in the grid

### Actual Result
![image](https://user-images.githubusercontent.com/39480008/67514593-9e32f380-f6ba-11e9-9fc3-90d73eceff87.png)

### Expected Result
![image](https://user-images.githubusercontent.com/39480008/67514713-e6521600-f6ba-11e9-8d8c-e8936e6af6b4.png)
